modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/main_view/props_transformer/cell_renderers/SimpleActionLinkRenderer.tsx

```diff
@@ -3,11 +3,15 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
+import ClayIcon from '@clayui/icon';
 import ClayLink from '@clayui/link';
+import classNames from 'classnames';
 import React from 'react';
 
 import formatActionURL from '../../../common/utils/formatActionURL';
+import {getFileMimeTypeValue} from '../utils/transformViewsItemProps';
 
+import '../../../../css/main.scss';
```

Comment:
I think that CSS import is not necessary, because it’s already being imported in `transformViewsItemProps`.

Reply:
Removed! ⭐